tabs/pid_tuning.js

```diff
@@ -8,9 +8,7 @@ var SUPEREXPO_FEATURE_BIT = 23;
 
 TABS.pid_tuning.initialize = function (callback) {
     var self = this;
-    if (GUI.active_tab != 'pid_tuning') {
-        GUI.active_tab = 'pid_tuning';
-    }
+    if (GUI.active_tab != 'pid_tuning') { GUI.active_tab = 'pid_tuning'; }
 
```

Comment:
Hmm… but these are NOT on separate lines. Or is there a typo?

Sure I can revert this, is there some kind of style guide? I mean unfortunately the whole project is pretty bad when it comes to software design and consistency. Really hope that will improve.